src/page-response-impact/PopulationReduction.tsx

```diff
@@ -0,0 +1,88 @@
+import hexAlpha from "hex-alpha";
+import React from "react";
+import styled from "styled-components";
+
+import Colors from "../design-system/Colors";
+
+const rightWidth = 65;
+
+const Container = styled.div`
+  max-width: 407px;
+`;
+const HeaderContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: 1px solid ${hexAlpha(Colors.forestGray, 0.2)};
+  font-weight: normal;
+`;
+const TitleGroup = styled.div`
+  font-family: "Poppins", sans-serif;
+  font-size: 13px;
+  line-height: 16px;
+`;
+const HeaderTitle = styled.span`
+  color: ${Colors.forest};
+`;
+const HeaderSubTitle = styled.span`
+  color: ${hexAlpha(Colors.forest, 0.5)};
+`;
+const Percent = styled.div`
+  font-size: 36px;
+  line-height: 1.5;
+  font-weight: 500;
+  color: ${hexAlpha(Colors.forest, 0.7)};
+  width: ${rightWidth}px;
```

Comment:
I think this might flow better with different sized percentages if you remove the set width here and max-width on the `Container` div

Reply:
good idea. I reworked with %s and seems pretty good.